common/src/main/java/org/tron/common/parameter/ResilienceConfig.java

```diff
@@ -0,0 +1,33 @@
+package org.tron.common.parameter;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j(topic = "net")
+public class ResilienceConfig {
+  @Getter
+  @Setter
+  private boolean enabled = false;
+
+  @Getter
+  @Setter
+  private int checkInterval = 60;
+
+  @Getter
+  @Setter
+  private int peerNotActiveThreshold = 600;
+
+  @Getter
+  @Setter
+  private int blockNotChangeThreshold = 300;
+
+  @Getter
+  @Setter
+  private boolean testStopInv = false;
```

Comment:
It is used to specify whether we test if the peer is malicious. There is alternative method. Only one of them can be used. It can be configured in config.conf.

Reply:
This is just a field in the rule. What I mean is that you should configure it according to the rules, not the fields in the rules. When you want to enable or disable rules, you can just change the configuration without changing the code.